examples/genomicsdb_query

```diff
@@ -57,7 +57,16 @@ def parse_callset_json_for_row_ranges(callset_file, samples=None):
     if isinstance(samples, str):
         with open(samples) as file:
             samples = [line.rstrip() for line in file]
```

Comment:
Can we make `samples` a set and then we don't need to do the nested loop below?

Reply:
Not fixed the nested loop issue, but fixed a bug for now. I will get to it soon.